### PR TITLE
Mutagenic turrets initial commit

### PR DESCRIPTION
**Pull request by lucastherrien**
## Proposed Changes
A thing I realized this mod lacked was mutagenic turrets. So, on a whim, I decided to make some. For now they are just a proof of concept, essentially the vanilla turrets just firing pawnmorpher projectiles.

The 3 turrets I've added are:

Mutagenic mini-turret: vanilla mini-turret that fires jellied mutanite balls
Mutagenic autocannon: vanilla autocannon firing jellied mutanite
Mutagenic cannon: A direct-fire artillery gun that fires mutagenic rounds with splash damage, and I'm gonna put the crude artwork I made for this on discord shortly after I post this

I'm probably going to work more on this after finals are done. First improvement that comes to mind is reloading them with slurry or mutanite instead of steel/uranium.

### CODE REVIEW

**Comment by mc-oofy** [comment]
i dont think this mod needs turrets

File: Defs/ThingDefs_Buildings/MutagenicTurrets.xml [hunk 1]
```diff
@@ -0,0 +1,457 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Defs>
+
+  <!-- Bases -->
+
+  <ThingDef Abstract="True" Name="BaseWeaponTurret" ParentName="BaseWeapon">
```
Comment (FxWolfxe) [inline Defs/ThingDefs_Buildings/MutagenicTurrets.xml:6]: These bases are included with vanilla and aren't needed here

**Comment by mc-oofy** [comment]
arent turrets sorta op since they are like rapidfire and you combine that with the fact that you can turn your enemy into a doggo

**Comment by mc-oofy** [comment]
pretty sure the bases were removed

**Comment by FxWolfxe** [comment]
This has been made into it's own mod

**Pull request closed without merging**